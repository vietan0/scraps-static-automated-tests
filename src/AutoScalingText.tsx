export default function AutoScalingText({ size }: { size: string }) {
  return (
    <div className={size} id="ASTbananas">
      AutoScalingText Hey
    </div>
  );
}
