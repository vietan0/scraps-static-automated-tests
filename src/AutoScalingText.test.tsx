import { render } from '@testing-library/react';
import { expect, test } from 'vitest';

import AutoScalingText from './AutoScalingText';

test('render', () => {
  const { container } = render(<AutoScalingText size="text-3xl" />);
  expect(container.firstChild).toMatchSnapshot();
});
