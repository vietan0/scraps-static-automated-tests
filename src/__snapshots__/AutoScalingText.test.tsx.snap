// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`render 1`] = `
<div
  class="text-3xl"
  id="ASTbananas"
>
  AutoScalingText Hey
</div>
`;
